modules/cli/src/integrationTest/java/org/apache/ignite/cli/ItConfigCommandTest.java

```diff
@@ -66,15 +70,21 @@ public class ItConfigCommandTest extends AbstractCliTest {
     private IgniteImpl node;
 
     @BeforeEach
-    void setup(@WorkDirectory Path workDir, TestInfo testInfo) {
-        node = (IgniteImpl) IgnitionManager.start(testNodeName(testInfo, 0), null, workDir);
+    void setup(@WorkDirectory Path workDir, TestInfo testInfo) throws NodeStoppingException {
+        String nodeName = testNodeName(testInfo, 0);
+
+        CompletableFuture<Ignite> future = IgnitionManager.start(nodeName, null, workDir);
+
+        IgnitionManager.init(nodeName, List.of(nodeName));
```

Comment:
I think we can add this option later if needed, since this API is experimental, and we do not yet have an agreement on how to implement the node start/init properly

Reply:
Ok